submissions/VikTorNik/html-css-popup/index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hooli-style Popup</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <div class="wrapper">
```

Comment:
Don't forget about mobile version.
![image](https://user-images.githubusercontent.com/16196199/190481243-45cc8d1e-db02-493b-ae66-0d202e4ff5a0.png)

Reply:
One more thing

![image](https://user-images.githubusercontent.com/16196199/192959597-4765aee8-f0dd-4cc2-b0b3-37cb9ee4189f.png)

Reply:
Fixed it.